src/main/java/uk/gov/hmcts/reform/em/orchestrator/service/caseupdater/CcdBundleStitchingService.java

```diff
@@ -84,11 +54,7 @@ private CcdValue<CcdBundleDTO> stitchBundle(CcdValue<CcdBundleDTO> bundle, CcdCa
             return bundle;
         } catch (InterruptedException e) {
             Thread.currentThread().interrupt();
-            throw new StitchingServiceException(cdamDto.getCaseId(), e.getMessage(), e);
+            throw new StitchingServiceException(e.getMessage(), e);
```

Comment:
Could you please revert this line change and leave the caseId in the Exception? Having caseId in the failure logs helps troubleshooting on Prod.